### PR TITLE
Docs: Overview

### PR DESCRIPTION
**Pull request by mariamera**
Closes [Ticket](https://trello.com)

Overview of the documentation. Please leave feedbacks to improve it 😃

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -9,8 +9,16 @@ Welcome to Paradise Paper Search App's Django + Neomodel Tutorial!
 
 Overview
 --------
-.. todo::
-   Here will be the tutorial overview.
+
+The aim of this tutorial is to demonstrate how to develop a web application using the Django framework backed by Neo4j, connecting them with the Neomodel driver. Neomodel is an Object Graph Mapper (OGM) for the Neo4j graph database.
+
+The tutorial covers several topics, some of the topics are: how to configure Neomodel within a Django program, create models, query to the database, create an API. It provides best-practice guidance on implementing Neomodel.
```
Comment (mariamera) [inline docs/index.rst]: tutorial covers several topics, some of them are

File: docs/index.rst [hunk 2]
```diff
@@ -9,8 +9,16 @@ Welcome to Paradise Paper Search App's Django + Neomodel Tutorial!
...
+
+The project is a web application to search for information on a Paradise Paper Graph Database. The data from the Database includes companies and people in more than 200 countries that are part of the Paradise Papers, Panama Papers, Bahama Leaks or the Offshore Leaks investigations.
+
+Each of these data records is displayed as a `Node` within the database. If you are not familiar with the definition of a node, it is a graphic data record. There is also a term that we are going to use called `relationship` which is what connects the nodes with each others. An example of these `relationships` could be ``REGISTERED_ADDRESS`` , ``OFFICER_OF``. What’s more, each node will have `properties` that are called data values.
+
```
Comment (oscarpolanco) [inline docs/index.rst]: A question here
Is `the nodes with each others` or `the nodes with each other`?
Sorry just want to know 😅

Comment (mariamera) [reply docs/index.rst]: each other :o you are right! good catch

**Review by oscarpolanco — APPROVED** [review]
LGTM

**Review by raulgomezc — APPROVED** [review]

File: docs/index.rst [hunk 3]
```diff
@@ -9,8 +9,45 @@ Welcome to Paradise Paper Search App's Django + Neomodel Tutorial!
...
+The aim of this tutorial is to demonstrate how to develop a web application using the Django 
+framework backed by Neo4j, connecting them with the Neomodel driver. Neomodel is an Object Graph 
+Mapper (OGM) for the Neo4j graph database.
+
+The tutorial covers several topics, some of them are: how to configure Neomodel within a Django 
+program, create models, query to the database, create an API. It provides best-practice guidance
+on implementing Neomodel.
+
+The project is a web application to search for information on a Paradise Paper Graph Database. 
+he data from the Database includes companies and people in more than 200 countries that are part
```
Comment (raulgomezc) [inline docs/index.rst]: Here I think there's a T missing

File: docs/index.rst [hunk 4]
```diff
@@ -9,8 +9,45 @@ Welcome to Paradise Paper Search App's Django + Neomodel Tutorial!
...
+The tutorial covers several topics, some of them are: how to configure Neomodel within a Django 
+program, create models, query to the database, create an API. It provides best-practice guidance
+on implementing Neomodel.
+
+The project is a web application to search for information on a Paradise Paper Graph Database. 
+he data from the Database includes companies and people in more than 200 countries that are part 
+of the Paradise Papers, Panama Papers, Bahama Leaks or the Offshore Leaks investigations.
+
+The search of the program can filter the mentioned data by country, jurisdiction and/or data source. 
+The approach adopted to make the program was to create a single-page web application, from which
```
Comment (raulgomezc) [inline docs/index.rst]: Think 'in which' sounds better than 'from which'. For some reason sounds off to me

File: docs/index.rst [hunk 5]
```diff
@@ -9,8 +9,45 @@ Welcome to Paradise Paper Search App's Django + Neomodel Tutorial!
...
+of the Paradise Papers, Panama Papers, Bahama Leaks or the Offshore Leaks investigations.
+
+The search of the program can filter the mentioned data by country, jurisdiction and/or data source. 
+The approach adopted to make the program was to create a single-page web application, from which 
+the data that is displayed will be obtained from an API. The API will fetch the data from the Neo4j 
+database and return it as a JSON. This allows you to retrieve fast segments of data in several 
+smaller requests, instead of making a single large request.
+
+The types of data you can encounter when searching are: Entity, Officer, Intermediary, Address,
+Other. Each of these data records is displayed as a `Node` within the graph database.If you are
```
Comment (raulgomezc) [inline docs/index.rst]: Missing space in '.If you are'

File: docs/index.rst [hunk 6]
```diff
@@ -9,8 +9,45 @@ Welcome to Paradise Paper Search App's Django + Neomodel Tutorial!
...
+not familiar with the definition of a node, it is a graphic data record. Here is an example: 
+
+.. image:: _static/node.png
+	:width: 100%
+
+In a graph database nodes are linked with each other with lines; they represent the relationship
+between them. This is call a `relationship`. On this application there are offshore entities that
```
Comment (raulgomezc) [inline docs/index.rst]: Sounds kinda redundant to say that the lines represent relationships and that they are called relationships

File: docs/index.rst [hunk 7]
```diff
@@ -9,8 +9,45 @@ Welcome to Paradise Paper Search App's Django + Neomodel Tutorial!
...
+have a registered address; Therefore there is a relationship type called `REGISTERED_ADDRESS`. 
+
+.. image:: _static/rel_type.png
+	:width: 100%
+
+On this example we can see a offshore entity register to an address on the bahamas.
```
Comment (raulgomezc) [inline docs/index.rst]: 'In this example' instead of 'On this example'

File: docs/index.rst [hunk 8]
```diff
@@ -9,8 +9,45 @@ Welcome to Paradise Paper Search App's Django + Neomodel Tutorial!
...
+
+What’s more, each node will have `properties` , which are essentially labels that are apply to
```
Comment (raulgomezc) [inline docs/index.rst]: Surplus space in '`properties` ,'

Comment (raulgomezc) [inline docs/index.rst]: 'labels that are applied' instead of 'apply'

File: docs/index.rst [hunk 9]
```diff
@@ -9,8 +9,45 @@ Welcome to Paradise Paper Search App's Django + Neomodel Tutorial!
...
+between them. This is call a `relationship`. On this application there are offshore entities that
+have a registered address; Therefore there is a relationship type called `REGISTERED_ADDRESS`. 
+
+.. image:: _static/rel_type.png
+	:width: 100%
+
+On this example we can see a offshore entity register to an address on the bahamas.
+
+What’s more, each node will have `properties` , which are essentially labels that are apply to 
+a record. For example, a Entity node might have a label country with the value of "Bahamas".
```
Comment (raulgomezc) [inline docs/index.rst]: ', an Entity' instead of ', a Entity'

File: docs/index.rst [hunk 10]
```diff
@@ -9,8 +9,45 @@ Welcome to Paradise Paper Search App's Django + Neomodel Tutorial!
...
+have a registered address; Therefore there is a relationship type called `REGISTERED_ADDRESS`. 
+
+.. image:: _static/rel_type.png
+	:width: 100%
+
+On this example we can see a offshore entity register to an address on the bahamas.
+
+What’s more, each node will have `properties` , which are essentially labels that are apply to 
+a record. For example, a Entity node might have a label country with the value of "Bahamas". 
+Having these labels allow the system to find all the nodes that are in "Bahamas", instead of the
```
Comment (raulgomezc) [inline docs/index.rst]: 'allows the system' instead of 'allow the system'

File: docs/index.rst [hunk 11]
```diff
@@ -9,8 +9,45 @@ Welcome to Paradise Paper Search App's Django + Neomodel Tutorial!
...
+ones that are in "Canada" or "Mexico". The properties can be compare as columns in SQl databases.
```
Comment (raulgomezc) [inline docs/index.rst]: 'can be compared to columns' instead of 'can be compare as columns'

**Review by raulgomezc — CHANGES_REQUESTED** [review]

File: docs/index.rst (hunk 6 above)
Comment (mariamera) [reply docs/index.rst]: I know, but that is how is called the term and that is what it does. or should I take down the `This is called a relationship`

**Review by raulgomezc — APPROVED** [review]

**Pull request merged**